src/routes/user.js

```diff
@@ -1,262 +1,254 @@
-import { Router } from 'express';
-import rateLimit from 'express-rate-limit';
-import validator from 'validator';
-import utils from '../utils';
-import email from '../email';
-
-const router = new Router();
-const max = (process.env.NODE_ENV !== 'production') ? 50000 : 5;
+import { Router } from 'express'
+import rateLimit from 'express-rate-limit'
+import validator from 'validator'
+import utils from '../utils'
+import email from '../email'
+
+const router = new Router()
+const max = (process.env.NODE_ENV !== 'production') ? 50000 : 5
```

Comment:
nit: can we give `max` a more specific name?

Reply:
I changed how it's used, so `max` is the best name here.